hazelcast/include/hazelcast/client/client_config.h

```diff
@@ -209,7 +217,7 @@ namespace hazelcast {
             *
             * \return loadBalancer
             */
-            load_balancer *const get_load_balancer();
+            std::unique_ptr<load_balancer> get_load_balancer();
```

Comment:
should we rather return reference here? With this version it looks more like  method like `move_load_balancer`. Consider that the user usually uses getters as read-only which is different from this usage.